test/test_examples_2d_advection.jl

```diff
@@ -36,41 +36,41 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
 
```

Comment:
Changing the domain_length naturally impacted the corresponding elixirs in their l2- and linf-errors.